csharp/property_test.go

```diff
@@ -16,6 +16,20 @@ MyType MyProperty
 	assertCode(t, property, expected)
 }
 
+func TestPropertyGetWithInitalizer(t *testing.T) {
+	expected := `
+string MyProperty
+{
+    get;
+}
+ = "foo";
```

Comment:
I had wanted 
```cs
string MyProperty
{
    get;
} = "foo";
```

but 🤷 I don't really feel like reworking a bunch of internal logic here if there is no easy way.

If we really want to pretty it up, I'd have it write 
```cs
string MyProperty { get; } = "foo";
```